### PR TITLE
Ukhsa immensa analysis

### PR DESCRIPTION
**Pull request by leoloman-dhsc**
Pull request to merge all the work undertaken

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+    │
+    ├── SECURITY.md        <- Security terms and conditions
+    │
+    └── README.md          <- The top-level README for developers using this project
+
+
+
+Replication
+----------
+This repository contains the code & data for the preprint publication (INSERT LINK HERE). The code is written in R, and data stored in csv format and a shape file will be called from the ONS api and stored as a geojson after the first run.
```
Comment (RobertASmith) [inline README.md]: insert link to medxiv preprint paper on this line

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+
+A master script `main_script.R` can be found in the main folder. To replicate the analysis and preprint run the script.
+
+This will run all of the scripts for each separate analysis discussed in the paper (e.g. descriptive plots, causal impact ... ).
+
+Extensions
+----------
+Data and code from this project have been published in the spririt of collaboration. Please feel free to replicate this work, or to adapt & develop the work further. When using the data and code from this repository please cite using the following.
+
+> Hounsome, L., et al. (2022). Epidemiological impact of a large number of incorrect negative SARS-CoV-2 test results in South West England during September and October 2021. https://doi.org/10.1101/2022.11.30.22282922
```
Comment (RobertASmith) [inline README.md]: Suggested alternative: "Hounsome, L., Herr, D., Bryant, R., Smith, R., Loman, L., Harris, J., Youhan, U., Dzene, E., Hadjipantelis, P., Long, H. and Laurence, T., 2022. Epidemiological impact of a large number of incorrect negative SARS-CoV-2 test results in South West England during September and October 2021. medRxiv, pp.2022-11."

**Comment by RobertASmith** [comment]
Hi @leoloman-dhsc,

Hope you are well. Congrats on getting this done.

Just checking whether you wanted me to review this, which may take a while so would have to wait a week or two. Assuming all checks have been done at UKHSA and Luke is happy for this to be made public I'm more than happy to just merge the request.

I've suggested two top level edits, just to add the link to the paper and a change in reference.

Kind regards,

Rob

File: main_script.R [hunk 3]
```diff
@@ -0,0 +1,113 @@
+# first time run to ensure fonts are working within R markdown
+# install.packages("remotes","extrafont")
```
Comment (RobertASmith) [inline main_script.R:2]: Recommend that you set up 'Renv' (https://rstudio.github.io/renv/articles/renv.html) to help with version control on packages. This has kind of taken over from 'pacrat' and is used to ensure that someone who runs this code in 5 years will use the same R package versions as we have here.

Here is an example for another open source project: https://github.com/RobertASmith/plumberHE/tree/main/renv, not sure if there is something similar in Python.

**Review by RobertASmith — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (leoloman-dhsc) [reply README.md]: Actioned

File: README.md (hunk 2 above)
Comment (leoloman-dhsc) [reply README.md]: Actioned

**Pull request merged**